dist/index.js

```diff
@@ -18,15 +18,15 @@ Object.defineProperty(exports, "__esModule", { value: true });
 const google_closure_compiler_1 = require("google-closure-compiler");
 const temp_write_1 = require("temp-write");
 const fs_1 = require("fs");
-function defaultCompileOptions(outputOptions) {
+exports.defaultCompileOptions = (outputOptions) => {
```

Comment:
Can we not gitignore dist?

Reply:
Good point, must have been left out earlier.